Core/scripts/dirac-install.py

```diff
@@ -2505,7 +2505,7 @@ def createSymbolicLink():
       logNOTICE("Writing down the releases files")
       releaseConfig.dumpReleasesToPath()
     logNOTICE("Installing modules...")
-    for modName in modsOrder:
+    for modName in set(modsOrder):
```

Comment:
From the name of the variable, I would assume that the order of entries matters, but a `set` does not preserve the order.